core/src/main/java/org/apache/iceberg/TableMetadataParser.java

```diff
@@ -277,6 +286,22 @@ static TableMetadata fromJson(FileIO io, InputFile file, JsonNode node) {
           schema, TableMetadata.INITIAL_SPEC_ID, node.get(PARTITION_SPEC)));
     }
 
+    JsonNode sortOrderArray = node.get(SORT_ORDERS);
+    List<SortOrder> sortOrders;
+    int defaultSortOrderId;
+    if (sortOrderArray != null) {
+      defaultSortOrderId = JsonUtil.getInt(DEFAULT_SORT_ORDER_ID, node);
+      ImmutableList.Builder<SortOrder> sortOrdersBuilder = ImmutableList.builder();
+      for (JsonNode sortOrder : sortOrderArray) {
+        sortOrdersBuilder.add(SortOrderParser.fromJson(schema, sortOrder));
+      }
+      sortOrders = sortOrdersBuilder.build();
+    } else {
+      SortOrder defaultSortOrder = SortOrder.unsorted();
```

Comment:
This variable doesn't seem necessary, but not a big deal.

Reply:
That var is used in two lines below, I think it is a bit easier to read if we preassign it.